app/Http/Controllers/Backoffice/Security/Permissions/PermissionsGetController.php

```
@@ -0,0 +1,158 @@
+            if (empty($roleId)) {
+                return new JsonResponse(
+                    [
+                        'title' => 'Bad Request',
+                        'status' => Response::HTTP_BAD_REQUEST,
+                        'detail' => 'Role ID is required'
+                    ],
+                    Response::HTTP_BAD_REQUEST
+                );
+            }
+
+            $response = ($this->searcher)(new PermissionsSearcherRequest($moduleId, (int) $roleId));
```

PHPStan (2.0.3)

Parameter #1 $moduleId of class MedineTech\Backoffice\Security\Permissions\Application\Search\PermissionsSearcherRequest constructor expects string, array|string given.


(argument.type)